frontend/src/components/DialogLayout/DialogLayout.scss

```diff
@@ -0,0 +1,56 @@
+@import "src/styles/variables";
+@import "src/styles/fonts";
+
+.dialogWindow {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  max-width: 720 * $px;
+  padding: 50 * $px;
+  margin-top: 100 * $px;
+
+  margin-right: auto;
+  margin-left: auto;
+  background-color: white;
+  border-radius: 10 * $px;
+
+  &__close {
+    position: absolute;
+    top: 40 * $px;
+    right: 40 * $px;
+    color: #9ba5a1;
+
+    &:hover {
+      color: #818a86;
+      cursor: pointer;
+    }
+  }
+
+  &__title {
+    @include headline1;
+  }
+
+  &__description {
+    @include paragraph-p2;
+    margin-top: 12 * $px;
+  }
+
+  &__spacer {
+    margin-bottom: 40 * $px;
+  }
+
+  &__line {
+    width: 100%;
+  }
```

Comment:
Looks like it do nothing

Reply:
like you? :putya:

Reply:
actually it's used by `CreateMethodologyDialog` and `CreateProjectDialog`

Reply:
I removed it and nothing was changed. Maybe I do not understand smth, please explain